sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.UnitTest/ConnectionString/ConnectionStringParserTests.cs

```diff
@@ -2,14 +2,13 @@
 // Licensed under the MIT License.
 
 using System;
-
-using NUnit.Framework;
+using Xunit;
```

Comment:
I would recommend changing the test framework as a separate PR. Your PR has gotten quite large and encapsulates several things not related to your original task.